src/components/App/functions.js

```diff
@@ -1,9 +1,7 @@
 import io from 'socket.io-client';
 
-const remoteHost = process.env.REMOTE_HOST || 'localhost';
-const remotePort = process.env.REMOTE_PORT || '8000';
-
-const socketPath = `http://${remoteHost}:${remotePort}`;
+const remoteHost = process.env.REMOTE_HOST || 'localhost:8000';
+const socketPath = `http://${remoteHost}`;
```

Comment:
now with SSL, shouldn't the host be using `https://`?

Reply:
Websocket automatically upgrades to SSL if `secure: false` is not set, and forcing this to HTTPS would bork dev environments without SSL certificates